canonical-web-and-design/maas.io
# Redirects to handle docs.maas.io URLs

When we go live, what we'll do is bulk redirect any requests to https://docs.maas.io/{path} to https://maas.io/docs/{path} instead.

This PR should do is make it so any URLs that exist on docs.maas.io will be redirected somewhere sensible.

QA
--

`./run clean && ./run`

Go to https://docs.maas.io and find some URLs. Now replace "docs.maas.io" with "127.0.0.1:8006/docs", and you should end up somewhere sensible.

E.g.:
- [ ] http://127.0.0.1/docs/2.2/en/installconfig-package-install should end up at https://old-docs.maas.io/2.2/en//installconfig-package-install
- [ ] http://127.0.0.1:8006/docs/2.6/en/ should end up at http://127.0.0.1:8006/docs/
- [ ] 127.0.0.1:8006/docs/2.6/en/intro-new should end up at http://127.0.0.1:8006/docs/whats-new-in-2-6
- [ ] Try out some of your own by browsing https://docs.maas.io
